release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2045,6 +2045,16 @@ In the table below, features are marked with the following statuses:
 
 * {product-title} 4.5 to {product-version} upgrade fails when an external network is configured on a bond device; the `ovs-configuration` service fails and nodes becomes unreachable. This will be resolved in a future 4.6.z release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887545[*BZ#1887545*])
 
+* Currently, HugePages are not detected properly when requested in several Non-Uniform Memory Access (NUMA) nodes. This is caused by the cnf-tests suite reporting errors when clusters contain several NUMA nodes because the tests compare the number of HugePages on one NUMA with the total number of HugePages on the entire node. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1889633[*BZ#1889633*])
+
+* The Data Plane Development Kit (DPDK) test used to check packet forwarding and receiving always fails. This will be resolved in a future 4.6.z release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1889631[*BZ#1889631*])
```

Comment:
Actually, all of these are planned for 4.6.z
Not sure this comment makes sense (or, if it does, it's worth extending to the others)

Reply:
Thanks, @fedepaol. I'll remove it there to avoid redundancy.

Reply:
@codyhoag can you please add the mention about 4.6.z to all issues added here, please?